cmd/fiovb.c

```diff
@@ -142,11 +142,69 @@ int do_fiovb_delete_pvalue(cmd_tbl_t *cmdtp, int flag, int argc,
 	return CMD_RET_FAILURE;
 }
 
+int do_is_secondary_boot(cmd_tbl_t *cmdtp, int flag, int argc,
+			 char * const argv[])
+{
+	int secondary;
+	char fiovb_val[2] = { 0 };
+
+	if (!fiovb_ops) {
+		printf("Foundries.IO Verified Boot is not initialized, run 'fiovb init' first\n");
+		return CMD_RET_FAILURE;
+	}
+
+	if (argc != 1)
+		return CMD_RET_USAGE;
+
+	if (fiovb_ops->secondary_boot(fiovb_ops, &secondary, false) ==
+	    FIOVB_IO_RESULT_OK) {
+		printf("Secondary boot bit = %d\n", secondary);
+		snprintf(fiovb_val, sizeof(fiovb_val), "%d", secondary);
```

Comment:
return value

Reply:
that should be fixed everywhere (including do_fiovb_read_pvalue(), do_fiovb_write_pvalue(), do_fiovb_delete_pvalue() etc)

Reply:
yes, but that doesnt make this lesss wrong does it?